Fallout2/Fallout1in2/mods/fo1_base/text/english/dialog/MASTER.MSG

```diff
@@ -117,17 +117,10 @@
 {300}{}{Duh!}
 
 
-# Master Tell-Me-Abouts generic response (You're to answer his questions, not vice versa. --BreakinBenny)
-{980}{Mast200}{I am the Master. I don't have to answer to you.}
-{981}{Mast201}{I am not here to answer your questions. You are here to answer mine.}
-
-#
-# Master Tell-Me-Abouts (Did I say he doesn't have any? :P --BreakinBenny)
-#
-
-{1000}{}{Unity}
-{1001}{}{Master}
-
-{1100}{Mast200}{I am the Master. I don't have to answer to you.}
-{1101}{Mast201}{I am not here to answer your questions. You are here to answer mine.}
+# Master Tell-Me-Abouts (You're to answer his questions, not vice versa. --BreakinBenny)
+{980}{}{Unity}
+{981}{}{Master}

```

Comment:
Now he will say Unity and Master whenever a keyword isn't found ...

Reply:
I like that! He may as well speak either voiceline because first, he doesn't have to answer any TMAs, and secondly because it gives some variety. (Though I feel the Master's TMA should be set up to not recognize any keywords at all, and thus always speak his TMA_MSG_DONTKNOW lines.)